fastapi_conn_db.py

```diff
@@ -0,0 +1,58 @@
+from fastapi import FastAPI
+from fastapi import Depends, HTTPException
+from sqlalchemy import create_engine
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import sessionmaker, Session
+from sqlalchemy import Column, ForeignKey, Integer, String
+
+DATABASE_URL = "mysql://root:rootroot11@localhost:3306/departments"
```

Comment:
This is sensitive data, I can see your root password. Moreover, I use another database user and password. Move this to .env file and create `.env.example`